styles/globals.css

```diff
@@ -2,25 +2,6 @@
 @tailwind components;
 @tailwind utilities;
 
-@layer base {
-  h1 {
-    @apply font-medium;
-  }
-
-  h2,
-  h3 {
-    @apply text-green-dark;
-  }
-
-  h2 {
-    @apply text-5xl leading-[67px];
-  }
-
-  h3 {
-    @apply text-4xl leading-[48px];
-  }
-}
```

Comment:
Note that I moved these to utility classes such as `.heading-1` so that we could have heading elements on other pages styled in different ways (for more semantically correct HTML etc.)